apps/files/src/components/BreadCrumbs.vue

```diff
@@ -132,6 +133,11 @@ export default defineComponent({
 		shouldShowBreadcrumbs(): boolean {
 			return this.filesListWidth < 768 && !this.isUploadInProgress
 		},
+
+		// used to show the views icon for the first breadcrumb
+		getViewIcon(): string | null {
+			return this.$navigation.active?.icon ?? null
+		}
```

Comment:
I do not think computed are getters, so I would go with `viewIcon` directly.
But how about simplify the code a but and use:

```suggestion
		getViewIcon(): string {
			return this.$navigation.active?.icon ?? homeSvg
		}
```

Then you can remove the `v-if` `v-else` in the template. And replace the import of `Home` with:
`import homeSvg from '@mdi/svg/svg/home.svg?raw'`

Reply:
Clean!